tests/test_api.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import numpy as np
+from libsvmdata import fetch_dataset, print_supported_datasets
+
+
+# TODO : add other datasets to test ?
+def test_replace():
+    X_first, y_first = fetch_dataset("iris", replace=True)
+    X_second, y_second = fetch_dataset("iris")
+    assert np.all(X_first == X_second)
+    assert np.all(y_first == y_second)
+
+
+def test_wrong_dataset():
+    with pytest.raises(ValueError):
+        fetch_dataset("unknowndataset")
+
+
+def test_print_supported_datasets():
+    print_supported_datasets()
+    assert True
```

Comment:
I'm not sure this is needed (honest question)

Reply:
I added that for better coverage but this is definitely a useless test

Reply:
I'm just talking about the assert. the rest is reasonable to have I believe

Reply:
Oh you're right, I though it necessary but apparently it is not